src/main/scan.ts

```diff
@@ -214,23 +202,16 @@ function isValidEventRow(rowData: CellData[]): boolean {
 }
 
 /**
- * Determines whether a given row has been crossed out (strikethrough) or not. A row is considered crossed out if all cells in the row that have
- * text have their strikethrough property set to true. The first Date objects in the first two columns are ignored as their strikethrough values
- * are always set as false according to the Google Apps Script api
+ * Determines whether a given row has been crossed out (strikethrough) or not. A row is considered crossed out if the WHAT cell has been crossed out
  * 
  * @param rowData data for this row
  * @returns boolean representing whether the given row has been crossed out or not
  */
 function isRowStrikethrough(rowData: CellData[]): boolean {
-    // Filters out all cells without any values
-    // Need to skip the time columns because their strikethrough values are incorrectly returned by the TextStyle object
-    const nonEmptyCells: CellData[] = rowData.filter((cellData, index) => cellData.value !== "" && index > END_TIME_COL_INDEX);
-
-    // Incrementally ands all of the strikethrough values together
-    return nonEmptyCells.every((cellData) => {
-        const runs: RichTextValue[] = cellData.richTextValue.getRuns();
-        return runs.every((run) => run.getTextStyle().isStrikethrough())
-    });
+    const whatCell: CellData = rowData[WHAT_COL_INDEX];
+    const whatCellRuns: RichTextValue[] = whatCell.richTextValue.getRuns();
+    // Incrementally ands all of the strikethrough values together to determine if the entire cell has strikethrough
+    return whatCellRuns.every((run) => run.getTextStyle().isStrikethrough());
```

Comment:
i'm assuming we'd want to replace every with some, right?
```suggestion
    return whatCellRuns.some((run) => run.getTextStyle().isStrikethrough());
```

Reply:
@andrewc5716 If we want to count the cell as having strikethrough when only part of the What cell has strikethrough then we should use `some`, but if we want to count the cell has having strikethrough when only the entire What cell has strikethrough then it should be `every`. Which do you prefer?

Reply:
i see, i thought it was possible for a run to exist if there was another style, like if it was both bold and strikethrough, so there'd be 2 runs in that case. or if part of it was bolded, there'd be a second smaller run? i guess i don't understand exactly how the runs get split up

Reply:
Hmm actually that's a good point. Let me test that case

Reply:
Switching to `some` to catch these edge run cases